pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -21,11 +21,14 @@
 , libXdmcp
 , fetchpatch
 , lndir
-, callPackages
+, callPackage
 
 , stable ? true
 , baseName ? "kicad"
-, versions ? { }
+, kicadSrc
+, kicadVersion
+, i18nSrc
+, i18nVersion
 , oceSupport ? false
 , opencascade
 , withOCCT ? true
```

Comment:
Using `*Support` variables and `with*` together is misleading, but it's an API that's already exposed. It could be deprecated rather nicely with an assertion as well (`lib.assertions.assertMsg` I think is good).